cuegui/cuegui/MenuActions.py

```
@@ -1739,6 +1740,80 @@ class HostActions(AbstractActions):
     def __init__(self, *args):
         AbstractActions.__init__(self, *args)
 
+    def canTakeOwnership(self, rpcObjects=None):
+        """Returns True when the selected host can be taken over."""
+        hosts = self._getOnlyHostObjects(rpcObjects)
+        return len(hosts) == 1 and hosts[0].data.lock_state == opencue.api.host_pb2.NIMBY_LOCKED
+
+    takeOwnership_info = ["Take Ownership", "Take ownership of a NIMBY-locked host", "configure"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)